src/python/grpcio_test/grpc_test/framework/interfaces/links/test_utilities.py

```diff
@@ -29,9 +29,26 @@
 
 """State and behavior appropriate for use in tests."""
 
+import logging
 import threading
+import time
 
 from grpc.framework.interfaces.links import links
+from grpc.framework.interfaces.links import utilities
+
+
+def _safe_ticket(ticket):
+  payload_length = -1 if ticket.payload is None else len(ticket.payload)
+  if ticket.payload is not None and 90 < payload_length:
```

Comment:
Almost completely arbitrary; I made a guess at what I thought would be not too painful to see printed in a log.